packages/taquito-core/src/error/errors.ts

```diff
@@ -1,20 +1,10 @@
 /**
  *  @category Error
- *  @description Error that indicates an invalid user data input
- */
-export class ExternalError extends Error {
-  public name = 'ExternalError';
-  constructor(public message: string) {
-    super(message);
-  }
-}
-
-/**
- *  @category Error
- *  @description Error that indicates an invalid user data input
+ *  @description Error that indicates the data get from other servies has incorrect format
```

Comment:
typo *services

Also I think instead of 'incorrect format' it could be more generalized as just 'invalid' since these errors might happen not only because of format errors.
i.e. 'Error that indicates invalid data being fetched from other services'